packages/types/src/index.ts

```diff
@@ -371,6 +371,12 @@ export type Leg = {
   serviceDate?: string;
   startTime: number | string;
   steps: Step[];
+  // Abridged version of what OTP returns
+  stopCalls?: {
+    stopLocation?: {
+      __typename?: string;
```

Comment:
Shouldn't `__typename` be a list of allowed string values? What are there cases where `__typename` would be not set in the OTP response?

Reply:
Good catch!